catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/java/org/codice/ddf/registry/admin/plugin/AdminRegistryPollerServiceBean.java

```diff
@@ -0,0 +1,302 @@
+                            registry.put(MAP_ENTRY_NAME, helper.getName(config));
+                            registry.put(MAP_ENTRY_BUNDLE_NAME, helper.getBundleName(config));
+                            registry.put(MAP_ENTRY_BUNDLE_LOCATION, config.getBundleLocation());
+                            registry.put(MAP_ENTRY_BUNDLE, helper.getBundleId(config));
+                        } else {
+                            registry.put(MAP_ENTRY_NAME, config.getPid());
+                        }
+
+                        Dictionary<String, Object> properties = config.getProperties();
+                        Map<String, Object> plist = new HashMap<>();
+                        for (String key : Collections.list(properties.keys())) {
+                            plist.put(key, properties.get(key));
+                        }
+                        registry.put(MAP_ENTRY_PROPERTIES, plist);
+
+                        configurations.add(registry);
+                    }
+                    metatype.put(MAP_ENTRY_CONFIGURATIONS, configurations);
+                }
+            } catch (Exception e) {
+                LOGGER.warn("Error getting registry info: {}", e.getMessage());
+            }
+        }
+
+        Collections.sort(metatypes,
+                (o1, o2) -> ((String) o1.get("id")).compareToIgnoreCase((String) o2.get("id")));
+        return metatypes;
+    }
+
+    @Override
+    public List<Map<String, Object>> allRegistryMetacards() {
+        List<Map<String, Object>> registryMetacardInfo = new ArrayList<>();
+
+        try {
+            List<RegistryPackageType> registryMetacardObjects =
+                    federationAdminService.getRegistryObjects();
+            for (RegistryPackageType metacardObject : registryMetacardObjects) {
+                registryMetacardInfo.add(RegistryPackageWebConverter.getRegistryObjectWebMap(
+                        metacardObject));
+            }
+        } catch (FederationAdminException e) {
+            LOGGER.warn("Couldn't get remote registry metacards '{}'", e);
+        }
+
+        return registryMetacardInfo;
+    }
+
+    protected AdminRegistryHelper getHelper() {
+        return new AdminRegistryHelper();
+    }
+
+    protected class AdminRegistryHelper {
+        protected ConfigurationAdmin configurationAdmin;
+
+        private BundleContext getBundleContext() {
+            Bundle bundle = FrameworkUtil.getBundle(AdminRegistryPollerServiceBean.class);
+            if (bundle != null) {
+                return bundle.getBundleContext();
+            }
+            return null;
+        }
+
+        protected List<Source> getSources() throws org.osgi.framework.InvalidSyntaxException {
+            List<Source> sources = new ArrayList<>();
+            List<ServiceReference<? extends Source>> refs = new ArrayList<>();
+            refs.addAll(helper.getBundleContext()
+                    .getServiceReferences(FederatedSource.class, null));
```

Comment:
Since we are in our own bean now i would change FederatedSource to RegistryStore